plugins/amazonq/codetransform/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codemodernizer/CodeWhispererCodeModernizerTestBase.kt

```diff
@@ -261,18 +273,20 @@ open class CodeWhispererCodeModernizerTestBase(
                     validManifest,
                     listOf(examplePatchVirtualFile),
                     validTransformationSummary,
+                    summaryFileMock,
```

Comment:
This is the only actual change I made in this file, other changes are just from the linter running